src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -1,5 +1,8 @@
-{
+﻿{
   "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
```

Comment:
I'll have to come back to do a full review, but this dependency is the first thing that sticks out. CLI (/cc @eerhardt) takes a dependency on this package and doesn't want the msbuild dependencies or `dnxcore50` import. I worked with this by making the tasks in the buildtools Tasks project, which references VersionTools. Is there a reason you can't do the same here?